encoding/descaling.md

```diff
@@ -1,477 +1,332 @@
 
-*Credits are usually done in 1080p*.
-
-There are various masks you can use
-to help with dealing with that issue,
-but it might be better
-to make use of existing wrappers instead.
-For this example I'll
-be using `inverse_scale` from `kagefunc`.
-
-```Py
-descaled = kgf.inverse_scale(src, height=878, kernel='bicubic', b=0, c=1/2, mask_detail=True)
+merged = core.std.ShufflePlanes([merged, clip], [0, 1, 2], vs.YUV)
 ```
 
-We can make use of the mask
-that `inverse_scale` uses internally
-as well.
+![](images/descale/descaled_symphoxv_credits_frame_non-masked.png)
 
-```Py
-descaled = kgf.inverse_scale(src, height=874, kernel='bicubic', b=0, c=1/2)
-descaled_mask = kgf._generate_descale_mask(vsutil.get_y(core.resize.Spline36(src, descaled.width, descaled.height)), vsutil.get_y(descaled), kernel='bicubic', b=0, c=1/2)
-```
+![A mask to catch the credits in Symphogear XV's OP](images/descale/descaled_symphoxv_credits_mask.png)
 
-![Kaguya-sama: Love Is War — OP (credits mask)](images/descale_credits_mask.png)
-
-![Kaguya-sama: Love Is War — OP (descaled)](images/descale_credits.png)
-
-Note that if you see the mask
-catching a lot of other stuff,
-you might want to consider *not* descaling
-that particular frame,
-or trying a different kernel/values.
-Chances are that you're either
-using the wrong kernel
-or that the frames you're looking at are native 1080p.
-
-![Manaria Friends — 01 (end card)](images/descale_native1080.png)
-
-![(Don't descale this)](images/descale_dontdescalethis.png)
-
-
-## Dealing with bad descaling
-
-There are various things you can do
-to deal with scenes that have issues
-even after descaling.
-`Eedi3` stands out in particular
-as a fantastic AA filter
-that really nails down bad lineart
-caused by bad descaling.
-It's at best a "cheat code",
-however.
-While it might fix up some issues,
-it won't fix everything.
-It's also incredibly slow,
-so you might want to
-[use it on just a couple of frames at a time](scenefiltering.md)
-rather than over the entire clip.
-
-Other than Eedi3,
-usually the results of bad descaling
-are so destructive that
-there isn't much you can do.
-If you have an encode that's
-badly descaled,
-you're better off
-finding a different one.
-If you've got bad results
-after descaling yourself,
-try a different kernel or values.
-Alternatively,
-try not descaling at all.
-
-At the end of the day,
-as mentioned in the introduction,
-you can't descale everything perfectly.
-Sometimes it's better to think of it
-as a magical anti-ringing/haloing/aliasing filter
-rather than a scaler.
-
-For example,
-here it was used
-specifically to fix up
-some bad aliasing
-in the source.
-
-![Akanesasu Shoujo — 01 (src)](images/descale_akanesasu_src.png)
-
-![Akanesasu Shoujo — 01 (rescaled)](images/descale_akanesasu_rescaled.png)
+![Merge the original credits with a re-upscaled frame (using nnedi3_resample)](images/descale/descaled_symphoxv_credits_frame_masked.png)
```

Comment:
I don't see the image here, but is it a composition or the final frame? In the latter case, the caption can't use imperative.

Reply:
I don't quite get what you mean by this. Could you elaborate?

Reply:
I think Fichte means the caption should sound more like `Original credits merged with a re-upscaled (nnedi3_resample) frame` instead of using a command-like phrase.